_data/utilities.yml

```diff
@@ -179,6 +179,12 @@ websites:
       img: nicorgas.png
       tfa: No
 
+    - name: O2
```

Comment:
It seems like they provide services in other countries as well as the UK. Please change the name to `O2 (UK)` as they seem to use different login systems. Thanks! 😄

Reply:
Done